e2e/test/scenarios/admin/datamodel/field.cy.spec.js

```diff
@@ -134,39 +133,42 @@ describe("scenarios > admin > datamodel > field", () => {
       cy.contains("Title");
     });
 
-    // [quarantined]: flake, blocking 3rd party PR
-    it.skip("allows 'Custom mapping' null values", () => {
+    it("allows 'Custom mapping' null values", () => {
+      const dbId = 2;
+      const remappedNullValue = "nothin";
+
       restore("withSqlite");
       cy.signInAsAdmin();
-      const dbId = 2;
+
       withDatabase(
         dbId,
-        ({ number_with_nulls: { num }, number_with_nulls_ID }) =>
-          cy.visit(
-            `/admin/datamodel/database/${dbId}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${number_with_nulls_ID}/field/${num}/general`,
-          ),
+        ({ NUMBER_WITH_NULLS: { NUM }, NUMBER_WITH_NULLS_ID }) => {
+          cy.request("GET", `/api/database/${dbId}/schemas`).then(
+            ({ body }) => {
+              const [schema] = body;
+
+              cy.visit(
+                `/admin/datamodel/database/${dbId}/schema/${dbId}:${schema}/table/${NUMBER_WITH_NULLS_ID}/field/${NUM}/general`,
+              );
+            },
+          );
+
+          cy.log("Change `null` to custom mapping");
+          cy.findByRole("heading", { name: "Display values" })
+            .closest("section")
+            .findByText("Use original value")
+            .click();
+          popover().findByText("Custom mapping").click();
+
+          cy.findByDisplayValue("null").clear().type(remappedNullValue).blur();
+          cy.button("Save").click();
+          cy.button("Saved!");
```

Comment:
We should add an assertion to avoid confusion. Using `testing-library` to find an element without acting or asserting on it is discouraged.
```suggestion
          cy.button("Saved!").should("be.visible");
```

Reply:
I'm actually in a group of people who prefer implicit assertions, but I hear you.
Good call.

Thank you <3

Reply:
➕, I'm fine with implicit assertions

Reply:
@uladzimirdev I really couldn't find where the discussion was, but explicit assertions have no disadvantages.

It could be ambiguous what an author intends to do with the element because there could be multiple things you could do with an element:
1. Assert that it's visible, not visible, not in the DOM, is in the DOM, etc.
2. Click it
3. type in it (if it's an input)
etc.

And I wouldn't know if there's no explicit action on an element whether you wanted to assert it or wanted to click it but forgot to add the `.click()` or something else.

In short, I still think it's worth being explicit with this.